src/schema/types.ts

```
@@ -1,6 +1,9 @@
 export type ProjectStatus = "all" | "started" | "on-hold" | "completed"
 export type TimeType = "day" | "days" | "week" | "month" | "year"
+import { z } from "zod"
 
+export type ProjectStatus = "all" | "started" | "on-hold" | "completed"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ProjectStatus'. Consider to delete it or rename it.

'ProjectStatus' is defined here:


(lint/suspicious/noRedeclare)